### DISCUSSION TITLE
인텔리제이 코드스타일 불일치에 대해서

### DISCUSSION THREAD

**Discussion by whxogus215** [Sejong-Java-Study/Gimme-Gonghak-Auth#30] [Ideas]
<img width="483" alt="image" src="https://github.com/user-attachments/assets/1aa78066-305a-4a3e-bb49-a08ccab1c315">

현재 인텔리제이의 코드스타일이 전부 다른 것 같아서 매번 develop 브랜치를 최신화할 때마다, import문의 배치가 변경되는 것 같습니다.
`Settings -> Editor -> Code Style -> Java`에서 상단의 `Scheme` 선택 창을 통해 현재 인텔리제이에 적용되어 있는 코드스타일을 확인할 수 있습니다. 다들 어떠한 스타일이 적용되어 있는지 댓글로 적어주세요!

**Comment by whxogus215** [comment]
<img width="542" alt="image" src="https://github.com/user-attachments/assets/976881a6-7b2e-474a-9d87-e313f866f9ee">

추가로 인텔리제이에서 코드의 마지막에 자동으로 개행을 추가하는 설정이 되어있는지도 확인해주세요!
`Settings -> Editor -> General -> On Save -> Ensure every saved file ends with a line break`

**Comment by gmltn9233** [comment]
![image](https://github.com/user-attachments/assets/7c9ecaea-970c-4e44-b841-0f803645dd98)
GoogleStyle로 적용되어있고 마지막 개행추가는 안되어 있어서 개행 추가하도록 수정하였습니다!

**Reply by whxogus215** [reply]
저는 현재 WootecoStyle로 되어있긴 한데, 다수결로 정하는게 좋을 것 같습니다!

**Reply by gmltn9233** [reply]
넵 좋습니다!

**Comment by jiseop121** [comment]
저도 동일하게 구글 코드 컨벤션을 사용하고 있어서 인텐션만 조정하겠습니다

**Comment by whxogus215** [comment]
넵 저도 구글 코드 컨벤션으로 변경하겠습니다
